### ISSUE TITLE
<use> tag should be void

### ISSUE THREAD

**Issue by sidouglas** [Prettyhtml/prettyhtml#106]
## 🐛 Bug Report

`<use>` tag should be void.

## To Reproduce

Given this markup
```html
 <svg
      class="o-svg loading-indicator__svg"
      focusable="false"
      role="img"
    >
      <use xlink:href="#svg-crescent"></use>
    </svg>
```
![bug](https://user-images.githubusercontent.com/1107946/55296410-c29c5980-5464-11e9-9ba6-a00a3923b443.png)

## Expected behavior
Expect to see
```
 <svg
       class="o-svg loading-indicator__svg"
      focusable="false"
      role="img"
    >
      <use xlink:href="#svg-crescent" />
    </svg>
```
![correct-output](https://user-images.githubusercontent.com/1107946/55296415-cc25c180-5464-11e9-8017-565d6f3e71e9.png)

## Your Environment

- Prettyhtml version: 0.8.16
- NodeJs version: v11.11.0
- Operating System and version: MacOS 10.14.3

**Comment by StarpTech** [comment]
Hi @sidouglas thank you for the bug report. Would you like to send a PR? The list of all the elements can be found [here](https://github.com/snabbdom/snabbdom-to-html/blob/9ecb7851ddf67bba4f0180e61f57aab429c7adce/elements.js).

**Comment by JayaKrishnaNamburu** [comment]
Hey, @StarpTech I too have the same issue with ``div`` tag. I am ok to raise a PR too, Please let me know 😄  and thanks for the great tool 😃 

<img width="1280" alt="Screenshot 2019-04-01 at 7 24 04 PM" src="https://user-images.githubusercontent.com/11075561/55332870-d4591d80-54b3-11e9-936d-807704553ba3.png">
<img width="1280" alt="Screenshot 2019-04-01 at 7 23 59 PM" src="https://user-images.githubusercontent.com/11075561/55332873-d4f1b400-54b3-11e9-9b34-a60eb3248633.png">

**Comment by StarpTech** [comment]
@JayaKrishnaNamburu don't see prettyhtml as the source of truth. In that case a div tag shouldn't be considered as void tag. We rely on [html-void-elements](https://github.com/wooorm/html-void-elements)

**Comment by JayaKrishnaNamburu** [comment]
@StarpTech was curious because ``use`` was also missing from the list, is there any way to mimic the behavior with the current settings?

**Comment by StarpTech** [comment]
@JayaKrishnaNamburu html-avoid-elements doesn't handle svg tags. For those use the other resource [here](https://github.com/snabbdom/snabbdom-to-html/blob/9ecb7851ddf67bba4f0180e61f57aab429c7adce/elements.js).

**Comment by JayaKrishnaNamburu** [comment]
Hey @StarpTech thanks for your time 👍

**Comment by JayaKrishnaNamburu** [comment]
Hey @StarpTech i would like to fix this PR, but I have a small doubt, since the void elements are being imported from [html-void-elements](https://github.com/wooorm/html-void-elements) I am not sure where to change this, since the same elements are passed down to the code, do you want me to append ``use`` to the list of elements that are being imported 🤔 😅

**Comment by sidouglas** [comment]
So does the buck get passed to the `html-void-elements` package?

**Comment by StarpTech** [comment]
Closed in #109
